qa/rpc-tests/cashlibtest.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+# Copyright (c) 2015-2018 The Bitcoin Unlimited developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+# This is a template to make creating new QA tests easy.
+# You can also use this template to quickly start and connect a few regtest nodes.
+
```

Comment:
"This is a template..."  should take out , maybe add something else or nothing at all?

Reply:
done